src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefaultCheckNullCaseLabel.java

```diff
@@ -0,0 +1,75 @@
+/*
+MissingSwitchDefault
+
+
+*/
+
+//non-compiled with javac: Compilable with Java21
+package com.puppycrawl.tools.checkstyle.checks.coding.missingswitchdefault;
+
+public class InputMissingSwitchDefaultCheckNullCaseLabel {
+    public void switchWithNullLabelTest(SwitchInput i) {
+        Integer counter=0;
+        switch (i) {
+            case FIRST: counter++;
+            case SECOND: counter--;
+            case null: counter=counter+2;
```

Comment:
We need to also have cases where the order of labels is permuted.

Reply:
Added test cases @nrmancuso